docs/atlascli/command/atlas-setup.txt

```diff
@@ -0,0 +1,114 @@
+.. _atlas-setup:
+
+===========
+atlas setup
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Register, authenticate, create and access an Atlas Cluster.
+
+This command takes you through registration, login, default profile creation, creating your first free tier cluster and connecting to it using MongoDB Shell.
```

Comment:
Question: do we want to mention it can also create a MongoDB user, add your IP address to the access list, and load sample data?

If it's helpful, here's the copy we are adding to the Atlas docs:

```
atlas setup performs all of the following steps:

      1. Signs up for an Atlas account.
      #. Creates one Atlas organization and one project.
      #. Creates one cluster in your Atlas project.
      #. Loads sample data into your Atlas cluster.
      #. Adds your IP address to your project's IP access list.
      #. Creates a MongoDB user for your Atlas cluster.
      #. Provides a connection string for your new cluster using
         the MongoDB Shell.
```

Reply:
After all, this part of the docs show up in the CLI, we ideally want to avoid having so much information in the command help description. 

I am fine with either option let's engage @dianchenghu for a final decision here.

Reply:
My concern for the detailed steps Sarah suggested might be overwhelming and cause the user to give up. I asked one user who tested with us before. He thinks both are OK. The first one is more friendly but second one is more helpful. In this case, the onboarding command is made for new users, we might lean toward the "friendly" a bit more. 
My suggestion is to keep the existing content.